client/app/scripts/liveblog-edit/controllers/blog-edit.js

```diff
@@ -265,6 +297,8 @@ define([
             },
             publish: function() {
                 $scope.actionPending = true;
+                //save the keep scoreres setting( if needed)
+                saveScorers();
```

Comment:
Here you should chain your promise to something. At least to verify that the save went well